apps/docker/docker-cmd.ts

```diff
@@ -38,11 +38,7 @@ shellCommandFactory(
 shellCommandFactory(
   "build",
   "Build the authserver and worldserver",
-  [
-    "docker compose --profile local --profile dev --profile dev-build build --parallel",
-    "docker image prune -f",
-    "docker compose run --rm --no-deps ac-dev-build bash apps/docker/docker-build-dev.sh",
-  ],
+  ["docker compose --profile app build"],
```

Comment:
`--parallel`  and the `docker image prune -f` can still be beneficial, ain't they?

Reply:
`--parallel` isn't un-helpful, but the build step only happens in one container and then the artifacts get copied to multiple containers, making that build step (which also takes the longest) a bottleneck. 

I'm pretty sure `docker compose build` and `docker compose up --build` actually are parallel by default? don't quote me on that :D

`docker image prune -f` can affect other containers/projects on the computer that I wouldn't want to mess with

Reply:
Apparently it isn't by default: https://ardalis.com/speed-up-docker-compose-with-parallel-builds/

You can measure the time with that option or without

Reply:
Going off of the documentation from docker compose, there is "unlimited" parallelism by default

https://github.com/docker/compose/blob/00f72cb553b6d35b6f3e18830ea8f2f5f785faac/docs/reference/compose.md?plain=1#L47

https://docs.docker.com/engine/reference/commandline/compose/#options